cpp/ql/src/Likely Bugs/ContinueInFalseLoop.qhelp

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>A <code>continue</code> statement only re-runs the loop if the loop-condition is true. Therefore using <code>continue</code> in a loop with a constant false condition will never cause the loop body to be re-run, which is is misleading.
```

Comment:
"loop-condition" > "loop condition"

Typo: "is is"

Reply:
Fixed.